src/main/java/org/folio/services/transactions/BaseTransactionService.java

```diff
@@ -24,8 +25,7 @@
 public class BaseTransactionService implements TransactionService {
   private static final Logger logger = LogManager.getLogger(BaseTransactionService.class);
   private static final int MAX_FUND_PER_QUERY = 5;
-  private static final String TRANSACTION_TO_QUERY = "(fiscalYearId==%s AND transactionType==%s) AND %s AND ((cql.allRecords=1 NOT fromFundId=\"\") OR %s)";
-  private static final String TRANSACTION_FROM_QUERY = "(fiscalYearId==%s AND transactionType==%s) AND %s AND ((cql.allRecords=1 NOT toFundId=\"\") OR %s)";
+  private static final String ALLOCATION_TYPE_TRANSACTIONS_QUERY = "(fiscalYearId==%s AND transactionType==%s) AND %s";
```

Comment:
Ideally, the query should be parameterized instead of using string substitution.

Reply:
I have been told this is actually for a CQL. Please disregard.